robot/pmm/tests/browser/ProgramEngagement/ProgramEngagement.robot

```diff
@@ -83,9 +82,9 @@ Date validation for PE when start date is later than end date
      Populate Modal Form                    Program Engagement Name= ${program_engagement_name}
      Populate Lookup Field                  Program                                ${program}[Name]
      Populate Lookup Field                  Client                                 ${contact}[FirstName] ${contact}[LastName]
-     Populate Lightning fields              Stage=Applied
-     ...                                    Role=Volunteer
-     ...                                    Start Date=25
+     Select Value From Dropdown             Stage                                  Applied
+     Select Value From Dropdown             Role                                   Volunteer
+     Populate Lightning fields              Start Date=25
      ...                                    End Date=10
```

Comment:
Should this and the line above be "Select From Date Picker" as in the next test?

Reply:
Fixed! Thanks for catching!